alembic/versions/2024_03_18_24c1e9e1476d_remove_unused_sample_columns.py

```diff
@@ -0,0 +1,29 @@
+"""Remove unused sample columns
+
+Revision ID: 24c1e9e1476d
+Revises: b2fcb2ada306
+Create Date: 2024-03-18 14:59:13.832047
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "24c1e9e1476d"
+down_revision = "b2fcb2ada306"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("sample", "sequence_start")
+    op.drop_column("sample", "is_external")
+    op.drop_column("sample", "invoiced_at")
```

Comment:
We have a single sample with this value set. Probably okay to just drop, but maybe good to check what that sample is about.

Reply:
Since we don't use the field for anything, does it matter? I'll check it though

Reply:
I think it is most likely fine. Seems to be from 2017